### PR TITLE
Add possibility to make screenshots of ParleyView

### PR DESCRIPTION
**Pull request by nlamah**
ParleyView is instantiated by default with the necessary object from the sharedInstance of the Parly-type. In the test target it is possible to use stubs, so that there is full control on the output on the view.

### CODE REVIEW

File: Sources/Parley/Views/ParleyTextView/ParleyTextView.swift [hunk 1]
```diff
@@ -5,19 +5,10 @@ final class ParleyTextView: UITextView {
...
-            boldFont = appearance.boldFont
             updateAttributedText()
         }
     }
 
-    @ParleyScaledFont(textStyle: .body) var linkFont = .systemFont(ofSize: 14)
-    @ParleyScaledFont(textStyle: .body) var regularFont = .systemFont(ofSize: 14)
-    @ParleyScaledFont(textStyle: .body) var italicFont = .italicSystemFont(ofSize: 14)
-    @ParleyScaledFont(textStyle: .body) var boldFont = .boldSystemFont(ofSize: 14)
-
```
Comment (alexkok) [inline Sources/Parley/Views/ParleyTextView/ParleyTextView.swift:20]: ❌ Regression:

These seemingly 'duplicated' fonts were needed to make the font scaling working while in-app. 

This change (removing these and putting them at `ParleyTextViewAppearance` breaks this realtime updating of the prefered size in the ParleyTextView.

Can be reproduced by opening the sample in the simulator, then press `CMD`+`Option`+`+` while the chat is open -- ParleyTextView won't automatically update it's font size anymore when removing this here.

**Review by alexkok — CHANGES_REQUESTED** [review]
Thanks for this improvement. 

On review we found one regression, please check that one. 

Rest looks good to merge. Great to support screenshot tests 👍

**Comment by nlamah** [comment]
> Thanks for this improvement.
> 
> On review we found one regression, please check that one.
> 
> Rest looks good to merge. Great to support screenshot tests 👍

This pull request will also give the possibility to make snapshots of the ParleyView with our appearance and discover quickly if there are regressions with version updates.

**Comment by alexkok** [comment]
Thanks, yeah the screenshots are great to test and verify with 👍

**Review by alexkok — APPROVED** [review]

**Pull request merged**